alertproduction.tex

```diff
@@ -25,10 +22,11 @@ \section{Alert Production in Commissioning and Early Operations}
 By the end of the commissioning period, coadd templates for use in difference imaging will only be available for $\approx$ 10\% of the sky.
 Generating templates over a wide area is not an explicit goal of commissioning;  however, where possible, if commissioning observations are agnostic to pointing and filter, we would endeavour to choose a pointing and filter that maximizes building templates to enable early science.
 
-Rubin aims to scale up alert production during commissioning with the aim of beginning routine Alert Production as soon as is feasible following System First Light  (\S~\ref{sec:timeline}).
+Rubin aims to scale up alert production during commissioning with the aim of beginning routine Alert Production as soon as is feasible following Rubin First Light  (\S~\ref{sec:timeline}).
```

Comment:
Maybe "... beginning routine Alert Production over a progressively increasing fraction of the sky as soon as feasible ..."?

Reply:
I'll apply the update